### PR TITLE
Support padding left

### PR DESCRIPTION
**Pull request by tastelikefeet**
# PR type
- [ ] Bug Fix
- [x] New Feature
- [ ] Document Updates
- [ ] More Models or Datasets Support

# PR information

Write the detail information belongs to this PR.

## Experiment results

Paste your experiment result here(if needed).

### CODE REVIEW

File: swift/torchacc_utils.py [hunk 1]
```diff
@@ -42,17 +42,18 @@ def _get_closet_bucket(bucket_sizes, data_length):
...
-                        world_size):
+                        world_size, padding_right):
     if padding_to is None:
         longest_len = input_ids.shape[-1]
         bucket_sizes = get_bucket_sizes(max_length)
         bucket_data_length = _get_closet_bucket(bucket_sizes, longest_len)
         padding_length = bucket_data_length - input_ids.shape[1]
-        input_ids = F.pad(input_ids, (0, padding_length), 'constant', tokenizer.pad_token_id)
-        attention_mask = F.pad(attention_mask, (0, padding_length), 'constant', 0)
+        pad_tuple = (0, padding_length) if padding_right else (padding_length, 0)
```
Comment (baoleai) [inline swift/torchacc_utils.py:51]: LGTM

**Review by baoleai — APPROVED** [review]

**Review by Jintao-Huang — APPROVED** [review]
